src/test/java/seedu/dengue/storage/SpecialisedStackForMemoryTest.java

```diff
@@ -0,0 +1,78 @@
+package seedu.dengue.storage;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.dengue.storage.temporary.SpecialisedStackForMemory;
+
+
+public class SpecialisedStackForMemoryTest {
```

Comment:
How come you testing with integers, should be possible to generate test cases using personbuilder to come up with meaningful states